src/normalizeUrl.js

```diff
@@ -0,0 +1,9 @@
+function normalizeUrl(requestUrl, requestHost) {
```

Comment:
Why do you create separate files for one function?
Just create 1 file like a `helpers` & put to this file all function for the server